### PR TITLE
In-Progress PR | New Features

### PR DESCRIPTION
**Pull request by EhsanFox**
There is a lot of stuff that I'm working on so, don't mind it yet.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -9,11 +9,11 @@ module.exports = (client) => {
 
     Events.CLICK_BUTTON = 'clickButton';
 
-    if (version < 11 || version === 11) {
+    if (version < 11 || version === 11)
```
Comment (es183923) [inline src/index.js:12]: why not just use 
```js
    if (version <= 11)
```

File: src/v12/Classes/MessageButton.js [hunk 2]
```diff
@@ -12,7 +12,7 @@ class MessageButton extends BaseMessageComponent {
 
     setup(data) {
 
-        this.style = 'style' in data ? resolveStyle(data.style) : null;
+        this.style = 'style' in data ? resolveStyle(data.style) : resolveStyle("blurple");
```
Comment (AngeloCore) [inline src/v12/Classes/MessageButton.js:15]: You should use [`MessageButtonStyles[]`](https://github.com/AngeloCore/discord-buttons/blob/main/src/v12/Constants.js#L3)

File: src/v13/Classes/MessageButton.js [hunk 3]
```diff
@@ -12,7 +12,7 @@ class MessageButton extends BaseMessageComponent {
 
     setup(data) {
 
-        this.style = 'style' in data ? resolveStyle(data.style) : null;
+        this.style = 'style' in data ? resolveStyle(data.style) : resolveStyle("blurple");
```
Comment (AngeloCore) [inline src/v13/Classes/MessageButton.js:15]: You should use [`MessageButtonStyles[]`](https://github.com/AngeloCore/discord-buttons/blob/main/src/v12/Constants.js#L3)

**Review by AngeloCore — CHANGES_REQUESTED** [review]
Styles

File: src/v12/Classes/MessageButton.js (hunk 2 above)
Comment (EhsanFox) [reply src/v12/Classes/MessageButton.js:15]: `MessageButtonStyles['blurple']`
????

Comment (AngeloCore) [reply src/v12/Classes/MessageButton.js:15]: `MessageButtonStyles.blurple`

Comment (AngeloCore) [reply src/v12/Classes/MessageButton.js:15]: [Import](https://github.com/AngeloCore/discord-buttons/blob/main/src/v12/Util.js#L1)
[Use](https://github.com/AngeloCore/discord-buttons/blob/main/src/v12/Util.js#L9)

Comment (EhsanFox) [reply src/v12/Classes/MessageButton.js:15]: Well `MessageButtonStyles['blurple']` is the same as `MessageButtonStyles.blurple` but ok, will do tomorrow

Comment (AngeloCore) [reply src/v12/Classes/MessageButton.js:15]: You're using `resolveStyle()`...

Comment (EhsanFox) [reply src/v12/Classes/MessageButton.js:15]: yea i will change to the one u said tho

File: src/index.js [hunk 4]
```diff
@@ -9,11 +9,11 @@ module.exports = (client) => {
...
     Events.CLICK_BUTTON = 'clickButton';
 
-    if (version < 11 || version === 11) {
+    if (version <= 11)
         throw new Error('The discord.js version must be v12 or high');
-    }
 
-    if (!client || !client instanceof Client) throw new Error("INVALID_CLIENT_PROVIDED: The discord.js client is not provided or is invalid...")
+    if (!client || !client instanceof Client)
+        throw new Error("INVALID_CLIENT_PROVIDED: The discord.js client is not provided or is invalid...")
```
Comment (es183923) [inline src/index.js:16]: no need for an ellipsis. Just use one period.

Comment (EhsanFox) [reply src/index.js:16]: good point

**Review by AngeloCore — APPROVED** [review]

File: src/index.js [hunk 5]
```diff
@@ -9,11 +9,11 @@ module.exports = (client) => {
 
     Events.CLICK_BUTTON = 'clickButton';
 
-    if (version < 11 || version === 11) {
+    if (version <= 11)
         throw new Error('The discord.js version must be v12 or high');
```
Comment (es183923) [inline src/index.js:13]: it should be `"The Discord.js version must be v12 or higher."`

**Comment by AngeloCore** [comment]
Wait a second, why we need default style?

**Comment by rayzdev** [comment]
> Wait a second, why we need default style?

It's not needed, but it makes some things easier because you don't have to specify a style if you want the button to be blurple anyways.

**Review by DeltaCoderr — APPROVED** [review]

**Pull request closed without merging**